src/accounts/context/userAccount.tsx

```diff
@@ -0,0 +1,97 @@
+// Libraries
+import React, {FC, useCallback, useEffect, useState} from 'react'
+// import {useParams, useHistory} from 'react-router-dom'
+import {useDispatch} from 'react-redux'
+
+// Types
+import {Account as UserAccount} from 'src/client/unityRoutes'
+
+// Utils
+import {getAccounts} from 'src/client/unityRoutes'
+
+export type Props = {
+  children: JSX.Element
+}
+
+export interface UserAccountContextType {
+  userAccounts: UserAccount[]
+  handleGetAccounts: () => void
+
+  defaultAccountId: number
+  activeAccountId: number
+}
+//   todo: add to above when implementing:
+//    setDefaultAccountId: (id: number) => void
+
+// isActive: true is for the currently logged in/active account
```

Comment:
what does this comment refer to? I don't see `isActive` as part of this component

Reply:
isActive is one of the flags returned with each 'account' object

Reply:
..moved to where the comment is relevant.